src/openhmd.c

```diff
@@ -491,6 +515,13 @@ void ohmd_set_default_device_properties(ohmd_device_properties* props)
 	props->ipd = 0.061f;
 	props->znear = 0.1f;
 	props->zfar = 1000.0f;
+	props->universal_distortion_k[0] = 0.0;
```

Comment:
So we already spoke about this, probably the best to set a universal standard for this (Generic/DK2 properties). Can this be used without the whole manual frustrum thing in the other drivers btw?